src/main/java/org/folio/circulation/domain/Account.java

```diff
@@ -27,9 +28,11 @@
   private final String status;
   private final String paymentStatus;
   private final Collection<FeeFineAction> feeFineActions;
+  private final DateTime createdAt;
```

Comment:
Some parts of the fee/fine logic uses the `createdAt` as the service point name where an account has been created. 

```suggestion
  private final DateTime creatioinDate;
```

Reply:
renamed